frontend/website/src/Developers.re

```diff
@@ -0,0 +1,160 @@
+let extraHeaders = () =>
+  <>
+    <script src="https://apis.google.com/js/api.js" />
+    <script
+      src="https://cdnjs.cloudflare.com/ajax/libs/marked/0.7.0/marked.min.js"
+      integrity="sha256-0Ed5s/n37LIeAWApZmZUhY9icm932KvYkTVdJzUBiI4="
+      crossOrigin="anonymous"
+    />
+  </>;
+
+module Styles = {
+  open Css;
+  let page =
+    style([
+      selector(
+        "hr",
+        [
+          height(px(4)),
+          borderTop(px(1), `dashed, Style.Colors.marine),
+          borderLeft(`zero, solid, transparent),
+          borderBottom(px(1), `dashed, Style.Colors.marine),
+        ],
+      ),
+    ]);
+
+  let header =
+    style([
+      display(`flex),
+      flexDirection(`column),
+      width(`percent(100.)),
+      color(Style.Colors.slate),
+      textAlign(`center),
+      margin2(~v=rem(3.5), ~h=`zero),
+    ]);
+
+  let content =
+    style([
+      display(`flex),
+      flexDirection(`columnReverse),
+      justifyContent(`center),
+      width(`percent(100.)),
+      marginBottom(`rem(1.5)),
+      media(Style.MediaQuery.somewhatLarge, [flexDirection(`row)]),
```

Comment:
May be worth double checking to see if our breakpoints match what Harold's specced out in figma